src/index.tsx

```diff
@@ -75,19 +70,7 @@ class Root extends PureComponent {
       <Provider store={getStore()}>
         <ConnectedRouter history={history}>
           <Suspense fallback={<PageSpinner />}>
-            <Switch>
-              {/* TODO(ariel): we need to restrict access to the checkout and operator pages based on specific critera:
-                https://github.com/influxdata/ui/issues/848
-               */}
-              {CLOUD && isFlagEnabled('unity-checkout') && (
-                <Route path="/checkout" component={CheckoutPage} />
-              )}
-              {/* Operator Routes */}
-              {CLOUD && isFlagEnabled('unity-operator') && (
-                <Route path="/operator" component={OperatorPage} />
-              )}
```

Comment:
Moved these further down the pipeline since they should only be accessible when a user is authenticated